about/quickstart.md

```diff
@@ -6,7 +6,7 @@ In this Docker quickstart guide, we'll get Ambassador Edge Stack running locally
 
 By default, Ambassador Edge Stack uses a demo configuration to show some of its basic features. Get it running with Docker, and expose Ambassador Edge Stack on port 8080:
 
-```shell
+```
```

Comment:
Seems like we probably shouldn't change this?

Reply:
8bf2b5d217e08e5c67644a0be1a692e8246c53bb

It seems that the shell syntax-highlighting was interfering with `$variable$ substitution :/